src/FPIndia/View.hs

```diff
@@ -53,12 +53,21 @@ renderBody rp model r = do
         renderMarkdown model "resources.md"
     renderFooter
 
-renderNavbar :: Prism' FilePath Route -> H.Html
-renderNavbar rp =
+renderNavbar :: Prism' FilePath Route -> HtmlRoute -> H.Html
+renderNavbar rp currentRoute =
   H.nav ! A.class_ "w-full h-1/4 text-xl font-bold flex space-x-4  mb-4" $ do
-    forM_ universe $ \r -> renderURL (H.toHtml $ routeTitle r) (routeHref rp r)
+    forM_ universe $ \r ->
+      if r == currentRoute
+        then renderURL (H.toHtml $ routeTitle r) (routeHref rp r) True
+        else renderURL (H.toHtml $ routeTitle r) (routeHref rp r) False
   where
-    renderURL menuItem path = H.a ! A.href path ! A.class_ "p-2 border-b-2 transition color hover:border-stone-900 duration-300" $ menuItem
+    renderURL menuItem path isActive =
```

Comment:
This function seems funky to me. 

If you were to add a function comment (not that we should) how would you explain "renderURL"? 

Can that all be made explicit in the function name and type?

Reply:
could rename to 

renderMenuItem title route isActive